### PR TITLE
Ijnek minor fixes against pr 1

### PR DESCRIPTION
**Pull request by ijnek**
Make all tests pass.
You have to enable CI in your fork to see them passing.

* Foxy -> Fails because building `Flova/geometry2-1:no_pykdl` fails on Foxy.
* Galactic -> Should succeed
* Rolling -> Fails due to [Python3.10 transition in Ubuntu Jammy](https://discourse.ros.org/t/preparing-for-rolling-sync-delayed-by-python-3-10-transition/24521/8)

### CODE REVIEW

**Comment by ijnek** [comment]
If Flova/geometry2-1:no_pykdl gets accepted into geometry2, we should remove the source installation of geometry2.
Unless the geometry2 changes get backported (which I'm sure it won't be), IPM won't be available in Foxy, and foxy CI should be removed.

**Comment by ijnek** [comment]
@Flova friendly ping on this. Please tell me if something isn't clear.

**Comment by Flova** [comment]
Sorry, I just forgot that this PR exists ^^

File: ipm_library/ipm_library/ipm.py [hunk 1]
```diff
@@ -14,15 +14,14 @@
 
 from typing import Optional
 
-import numpy as np
-import tf2_ros
+from ipm_library import utils
+from ipm_library.exceptions import NoIntersectionError
 from ipm_msgs.msg import PlaneStamped
+import numpy as np
```
Comment (Flova) [inline ipm_library/ipm_library/ipm.py:20]: Is this import formatting standard in ROS2? I find it kind of weird. Is there some guideline for this? In most python projects, they are grouped by `from` and `import`.

**Review by Flova — APPROVED** [review]
Could be that there are some changes that added `"` to the original PR, so we need to crosscheck this, but it otherwise lgtm.

File: ipm_library/ipm_library/ipm.py (hunk 1 above)
Comment (ijnek) [reply ipm_library/ipm_library/ipm.py:20]: Ah yes, the import orders were juggled around in  https://github.com/Flova/ipm/pull/1/commits/e38d324180e750b7777e80f58976e12b54ad17e6 to make flake8 happy but I don't really understand why this makes flake8 happy. 

From memory, [flake8: import statements are in the wrong order](https://stackoverflow.com/questions/39739118/flake8-import-statements-are-in-the-wrong-order) was the exact issue that arose and the suggested fix in that link says to add a .flake8 file to configure flake8-import-order plugin is needed.

I couldn't get this working, but juggling around the import orders did fix the issue. Would you know much about the fix suggested on StackOverflow......?

Comment (ijnek) [reply ipm_library/ipm_library/ipm.py:20]: Moved conversation to https://github.com/ros-sports/ipm/pull/1#discussion_r830680029

**Pull request merged**